src/app/components/confirmBtcTransactionComponent/index.tsx

```diff
@@ -280,7 +280,18 @@ function ConfirmBtcTransactionComponent({
     feeRate?: string;
     nonce?: string;
   }) => {
-    setCurrentFee(new BigNumber(modifiedFee));
+    const newFee = new BigNumber(modifiedFee);
+
+    if (
+      feeMultipliers &&
+      newFee.isGreaterThan(new BigNumber(feeMultipliers.thresholdHighSatsFee))
+    ) {
+      setShowFeeWarning(true);
```

Comment:
we also want to show the warning if the initial fee from transaction is greater than the threshold.

Reply:
I updated the code so now it should show it for the initial fee as well